### DISCUSSION TITLE
ImportError: cannot import name 'Invertd' from 'monai.transforms'

### DISCUSSION THREAD

**Discussion by ramchandracheke** [Project-MONAI/MONAI#2281] [General]
Hi,

I am facing issue while importing Invertd transform. Rest of the transforms are importing perfectly. 

![image](https://user-images.githubusercontent.com/46112629/120088385-ca5cf480-c0e7-11eb-8253-031915d30fcd.png)

Can you please help in this case. Thanks a million.

**Comment by Nic-Ma** [comment]
Hi @ramchandracheke ,

May I know your MONAI version?
`Invertd` is newly added in the master branch, so please use the latest weekly-build for this feature.

Thanks.

**Reply by ramchandracheke** [reply]
Thanks @Nic-Ma. 

Using '''pip install monai-weekly''', now the error is resolved now.
